README.md

```diff
@@ -54,9 +54,9 @@
 
```

Comment:
- Hi, Can you move to the license section to set MIT license? I am asking this because the requirement recommends using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
- This will make your Readme template more professional.

Reply:
Thank you very much for this reminder! I completely overlook the LICENSE, thinking I added it with the README